Calendar/Calendar.php

```diff
@@ -347,7 +347,7 @@ function returnValue($returnType, $format, $stamp, $default)
             break;
         case 'object':
             include_once CALENDAR_ROOT.'Factory.php';
-            return Calendar_Factory::createByTimestamp($returnType, $stamp);
+            return (new Calendar_Factory)->createByTimestamp($returnType, $stamp);
```

Comment:
Probably better coding style to just initiate a $factory variable; less chance of really old BC breaks